ibm/mas_devops/playbooks/mas/configure-app.yml

```diff
@@ -7,7 +7,7 @@
 
     # MAS workspace configuration
     mas_workspace_id: "{{ lookup('env', 'MAS_WORKSPACE_ID') }}"
-
+    manage_demo_data: "{{ lookup('env', 'MAS_DB_IMPORT_DEMO_DATA') | default( 'false' , true) }}"
```

Comment:
Is this var described anywhere?

Reply:
manage_demo_data is added under vars/defaultspecs/manage.yml
and moved the variable under default folder